backend/inspirehep/records/views.py

```diff
@@ -48,6 +51,28 @@ def get(self, pid, record):
         return jsonify(data)
 
 
+class LiteratureReferencesResource(MethodView):
+    view_name = "literature_references"
+
+    @pass_record
+    def get(self, pid, record):
+        page = request.values.get("page", 1, type=int)
+        size = request.values.get("size", 25, type=int)
+
+        if page < 1 or size < 1:
+            abort(400)
+
+        references = record.get("references", [])
+        selected_references = references[(page - 1) * size : page * size]
```

Comment:
if we allow any page and size this might cause ``500`` we should catch this and display with a migniful message or even better restrict the values